includes/init_includes/init_https.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+if (((HTTPS_SERVER == HTTP_SERVER) || (IS_ADMIN_FLAG)) && (strpos(HTTP_SERVER, 'https:') !== false)) {
```

Comment:
Since Admin doesn't technically define HTTPS_SERVER, switching the order of these would save a constant-not-defined error:

```diff
-if (((HTTPS_SERVER == HTTP_SERVER) || (IS_ADMIN_FLAG)) && (strpos(HTTP_SERVER, 'https:') !== false)) {
+if ((IS_ADMIN_FLAG || HTTPS_SERVER == HTTP_SERVER) && (strpos(HTTP_SERVER, 'https:') !== false)) {
```
Also, removing unnecessary parentheses (as shown) makes it more readable.